weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallsPage/CallsTableButtons.tsx

```diff
@@ -666,34 +667,75 @@ export const PaginationButtons = () => {
   const end = Math.min(rowCount, (page + 1) * pageSize);
 
   return (
-    <Box display="flex" alignItems="center" justifyContent="center" padding={1}>
-      <Button
-        variant="quiet"
-        size="medium"
-        onClick={handlePrevPage}
-        disabled={page === 0}
-        icon="chevron-back"
-      />
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
+      padding={1}
+      width="100%">
+      <Box display="flex" alignItems="center">
+        <Button
+          variant="quiet"
+          size="small"
+          onClick={handlePrevPage}
+          disabled={page === 0}
+          icon="chevron-back"
+        />
+        <Box
+          mx={1}
+          sx={{
+            fontSize: '14px',
+            fontWeight: '400',
+            color: MOON_500,
+            // This is so that when we go from 1-100 -> 101-200, the buttons dont jump
+            minWidth: '90px',
+            display: 'flex',
+            justifyContent: 'center',
+          }}>
+          {start}-{end} of {rowCount}
+        </Box>
+        <Button
+          variant="quiet"
+          size="small"
+          onClick={handleNextPage}
+          disabled={page >= pageCount - 1}
+          icon="chevron-next"
+        />
+      </Box>
       <Box
-        mx={1}
-        sx={{
-          fontSize: '14px',
-          fontWeight: '400',
-          color: MOON_500,
-          // This is so that when we go from 1-100 -> 101-200, the buttons dont jump
-          minWidth: '90px',
-          display: 'flex',
-          justifyContent: 'center',
-        }}>
-        {start}-{end} of {rowCount}
+        display="flex"
+        alignItems="center"
+        marginLeft="auto"
+        marginRight={1}
+        sx={{fontSize: '14px', color: MOON_500}}>
+        <Box
+          component="span"
+          sx={{
+            fontSize: '12px',
+            border: `1px solid ${MOON_400}`,
+            fontWeight: '600',
+            padding: '0px 4px',
+            marginRight: '4px',
+            borderRadius: '4px',
+          }}>
+          {isMac ? 'Option' : 'Alt'}
+        </Box>
+        +
+        <Box
+          component="span"
+          sx={{
+            fontSize: '12px',
+            border: `1px solid ${MOON_400}`,
+            fontWeight: '600',
+            padding: '0px 4px',
+            marginLeft: '4px',
+            marginRight: '4px',
+            borderRadius: '4px',
+          }}>
+          Click
+        </Box>
+        on a cell to filter by the value
```

Comment:
nit: this could be its own decomposed component. name suggestions: "CellSortCallout", "PaginationControlsSortCallout" etc